propagator/xray/lib/opentelemetry/propagator/xray/version.rb

```diff
@@ -7,7 +7,7 @@
 module OpenTelemetry
   module Propagator
     module XRay
-      VERSION = '0.17.0'
+      VERSION = '0.18.0'
```

Comment:
```suggestion
      VERSION = '0.17.0'
```
You don't need to bump this - it'll be bumped automatically during release.

Reply:
unbumped